scripts/spell_check/spelling.dat

```diff
@@ -2390,6 +2390,8 @@ dosent':doesn't
 dosen't:doesn't
 doub:doubt
 doulbe:double
+double click:double-click
+doubleclick:double-click
```

Comment:
well, if it works like that...Can't tell without digging a bit more.
Thanks a lot for testing!